documentation/tutorial.md

```diff
@@ -1,28 +1,37 @@
-# Charmed MongoDB tutorial:
-Ready to try out Charmed MongoDB? This tutorial will guide you through the steps to get Charmed MongoDB up and running. 
+# Charmed MongoDB tutorial
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform on top of juju. As a first step this tutorial show you how to get Charmed MongoDB up and running, but the tutorial does not stop there. Through this tutorial you will learn a variety of operations, everything from adding replicas to advanced operations such as enabling TLS. Specifically this tutorial will cover:
+- Setting up your environment using LXD and Juju.
+- Deploying MongoDB using a single command.
+- Accessing the admin database directly.
+- Adding high availability with replication.
+- Changing the admin password.
+- Automatically creating users with relations. 
+- Enabling secure transactions with TLS.
 
-## Minimum requirements:
+
+## Minimum requirements
 Before we start be sure you have the following requirements:
 - Ubuntu 20.04(focal) or later.
 - 8GB of RAM.
 - 2 CPU threads.
 - At least 20GB of available storage.
 - Access to the internet for downloading the required snaps and charms.
 
-## Install and prepare LXD:
+
+## Install and prepare LXD
 The fastest, simplest way to get started with Charmed MongoDB is to set up a local LXD cloud. LXD is a system container and virtual machine manager; Charmed MongoDB will be run in one of these containers and managed by Juju. While this tutorial covers the basics of LXD, you can explore more LXD [here](https://linuxcontainers.org/lxd/getting-started-cli/). The first step on our journey is to install LXD. LXD is installed from a snap package:
-```
+```shell
 sudo snap install lxd --classic
 ```
 
 Next we need to run `lxd init` to perform post-installation tasks. For this tutorial the default parameters are preferred and the network bridge should be set to have no ipv6 addresses:
```

Comment:
I already had lxd installed from a previous tutorial - so it wouldn't let me install classic. 

I'm not sure how to tell if the following two commands completed successfully, other than the fact that there were no errors?

Reply:
Is it worth mentioning why the network bridge should have no IPv6 address? Or can it be assumed that this is something your users would already know?

Reply:
LXD comes on 20.04 or later, so this part of the tutorial should be updated, let me know what you think of these changes.

"Is it worth mentioning why the network bridge should have no IPv6 address? Or can it be assumed that this is something your users would already know?" -- yes, but I am not sure what exactly to write here, I blatantly copy pasted this from "Expand to see the instructions for LXD" on the [Juju LXD tutorial](https://juju.is/docs/olm/get-started-with-juju):
```
LXD should be already installed in your Ubuntu system. Execute the code below to configure it:

lxd init --auto
lxc network set lxdbr0 ipv6.address none
```
Let me see if I can figure out why we do this

Reply:
apparently LXD wth ipv6 does not work with Juju - [source](https://discourse.charmhub.io/t/juju-and-ipv6/2210), I will add something to this effect